pkg/cmd/allocsim/main.go

```diff
@@ -496,7 +499,8 @@ func main() {
 		os.Exit(exitStatus)
 	}()
 
-	c.Start("allocsim", *workers, os.Args[0], nil, flag.Args(), perNodeArgs)
+	allNodeArgs := append(flag.Args(), "--vmodule=allocator=1")
+	c.Start("allocsim", *workers, os.Args[0], nil, allNodeArgs, perNodeArgs)
```

Comment:
Note that you can pass arguments through to the sub-processes by running `allocsim` like:

```
allocsim -- --vmodule=allocator=1
```

So perhaps this part of the change isn't necessary.